Source/Android/app/src/main/res/values/strings.xml

```diff
@@ -841,6 +841,23 @@ It can efficiently compress both junk data and encrypted Wii data.
     <string name="about_github"><a href="https://github.com/dolphin-emu/dolphin">GitHub</a></string>
     <string name="about_support"><a href="https://forums.dolphin-emu.org/">Support</a></string>
     <string name="about_copyright_warning">\u00A9 2003–2015+ Dolphin Team. \u201cGameCube\u201d and \u201cWii\u201d are trademarks of Nintendo. Dolphin is not affiliated with Nintendo in any way.</string>
+    <string name="system_driver">System driver</string>
+    <string name="system_driver_desc">The GPU driver that is part of the OS.</string>
+
+    <!-- Custom GPU drivers -->
+    <string name="gpu_driver_dialog_title">Select the GPU driver that Dolphin is going to use!</string>
```

Comment:
This string is a bit too long (and a bit too informal, especially with the ! at the end) for a popup title. Could you condense it, or at least remove the ! at the end?

Reply:
Changed it to "Select the GPU driver for Dolphin".